manifests/0000_90_dns-operator_03_prometheusrules.yaml

```diff
@@ -19,14 +19,16 @@ spec:
         labels:
           severity: warning
         annotations:
-          message: "{{ $value }} CoreDNS panics observed on {{ $labels.instance }}"
+          summary: CoreDNS panic
+          description: "{{ $value }} CoreDNS panics observed on {{ $labels.instance }}"
```

Comment:
Does `description` supersede `message`?  The alerting-consistency enhancement document doesn't say whether `message` is still needed or not, so if you don't know the answer, we can ask the document's author for clarification or examples.

Reply:
Good point, I used the cluster-etcd-operator as an example, they did not have 'message'. I will look into the document and ask if need be.

Reply:
The documentation mentions this 

**Documentation Required**

```
1. The name of the alerting rule should clearly identify the component impacted by the issue (for example etcdInsufficientMembers instead of InsufficientMembers, MachineConfigDaemonDrainError instead of MCDDrainError). It should camel case, without whitespace, starting with a capital letter. The first part of the alert name should be the same for all alerts originating from the same component.
2. Alerting rules should have a "severity" label whose value is either info, warning or critical (matching what we have today and staying aside from the discussion whether we want minor or not).
3. Alerting rules should have a description annotation providing details about what is happening and how to resolve the issue.
4. Alerting rules should have a summary annotation providing a high-level description (similar to the first line of a commit message or email subject).
5. If there's a runbook in https://github.com/openshift/runbooks, it should be linked in the runbook_url annotation.```

So it looks like `summary` & `description` are mandatory. I will ask about message

Reply:
From the #forum-monitoring _"it hasn't been superseded, but there was a consensus upstream to use summary and description in favor of message in order to have multiple level of abstraction"_ this is what upstream recommends: https://github.com/monitoring-mixins/docs#guidelines-for-alert-names-labels-and-annotations. So I think we can drop the message annotation.